src/playlist-loader.js

```diff
@@ -251,6 +251,8 @@ export default class PlaylistLoader extends EventTarget {
       customTagMappers: this.customTagMappers
     });
 
+    playlist.lastRefresh = Date.now();
```

Comment:
It looks like this is updated both in this function and within a different XHR request below. It may be worth centralizing it by creating a `requestManifest` wrapping function that takes in xhr options and a callback. It may also make testing easier.

Reply:
I think this one is a bit less necessary than the one below, I even considered removing it, but there's one codepath that leads here that isn't from the XHR below.
If it's useful for testing I'll add it. Otherwise, since it's a single line and only in two places, I think it's fine to leave as is. If we'd need it in another place then definitely worth refactoring it out.